observability/readme.md

```diff
@@ -37,7 +37,50 @@ In addition to above, engineers are encouraged to have conversations to understa
 
 ## Recommended Tools and Approaches
 
-1. [Application Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview)
+### Tools
+
+1. [Azure Monitor](https://docs.microsoft.com/en-us/azure/azure-monitor/overview) - Umbrella of services including system metrics, log analytics and more.
+2. [Application Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview) - Eventually will be included inside Azure Monitor.
+3. [Azure Log Analytics](https://docs.microsoft.com/en-us/azure/azure-monitor/log-query/log-query-overview) - Run advanced queries against the data collected via Azure Monitor Logs. 
+
+### Collection 
+
+There are many ways to collect data; here are three generally accepted methods:
+
+#### 1. Agent: 
+
+In most cloud-oriented applications, logs and other types of telemetry data are treated as streams of data; sometimes, those streams might be accessible as files, or something else. Many times, it would be STDOUT. An agent collector is executed as a daemon, a standalone application, or a container, and consumes these streams and forwards them to the system of record.
```

Comment:
```suggestion
In most cloud-oriented applications, logs and other types of telemetry data are treated as streams of data; sometimes, those streams might be accessible as files, or something else. Many times, it would be [`STDOUT`](https://en.wikipedia.org/wiki/Standard_streams). An agent collector is executed as a daemon, a standalone application, or a container, and consumes these streams and forwards them to the system of record.
```

Reply:
Made the changes suggested.